RxTestExt/Classes/RxTextExtensions.swift

```diff
@@ -2,57 +2,46 @@ import RxTest
 import XCTest
 
 extension TestableObserver {
-
-    public func assertErrorIs<T: Error>(_ type: T, file: StaticString = #file, line: UInt = #line) {
-        if let error = getErrorSafely(file: file, line: line) {
-            XCTAssert(error.self is T, file: file, line: line)
-        }
-    }
-
-    public func assertThatError(that: @escaping (Error) -> Bool, file: StaticString = #file, line: UInt = #line) {
-        if let error = getErrorSafely(file: file, line: line) {
-            XCTAssertTrue(that(error), file: file, line: line)
-        }
-    }
-
-     public func assertNoValues(_ file: StaticString = #file, line: UInt = #line) {
+    
+    // MARK: Value assertions
+    public func assertNoValues(_ file: StaticString = #file, line: UInt = #line) {
         XCTAssertEqual(0, valueCount, file: file, line: line)
     }
-
-     public func assertValueCount(_ count: Int, file: StaticString = #file, line: UInt = #line) {
+    
+    public func assertValueCount(_ count: Int, file: StaticString = #file, line: UInt = #line) {
         XCTAssertEqual(count, valueCount, file: file, line: line)
     }
-
-     public func assertThatAt(_ index: Int, that: @escaping(Element) -> Bool, file: StaticString = #file, line: UInt = #line) {
+    
+    public func assertThatAt(_ index: Int, that: @escaping(Element) -> Bool, file: StaticString = #file, line: UInt = #line) {
         if let element = getElementSafely(index, file: file, line: line) {
             XCTAssertTrue(that(element))
         }
     }
-
+    
     public func assertComplete(file: StaticString = #file, line: UInt = #line) {
         XCTAssertNotEqual(0, completions, file: file, line: line )
     }
-
+    
     public func assertNotComplete(file: StaticString = #file, line: UInt = #line) {
         XCTAssertEqual(0, completions, file: file, line: line )
     }
-
+    
     public func assertErrorMessage(message: String, file: StaticString = #file, line: UInt = #line) {
         if let error = getErrorSafely(file: file, line: line) {
             XCTAssertEqual(error.localizedDescription, message, file: file, line: line)
         }
     }
-
+    
     // MARK: internal functions
-
+    
     internal func getElementSafely(_ index: Int, file: StaticString, line: UInt ) -> Element? {
```

Comment:
I don't think the name of this function makes it clear that the test will fail if couldn't get the element safely.

Reply:
I have renamed to `getElementOrFail` would you think that was clearer?